lib/foods.js

```diff
@@ -1,65 +1,75 @@
 $('document').ready(function() {
+  for (var i = 0; i < localStorage.length; i++){
+    $("tbody").append(localStorage.getItem(localStorage.key(i)));
```

Comment:
@wlffann yes ma'am because when we add the item into local storage it is set up as  the foodRow with all the html